### PR TITLE
Exercise 1

### PR DESCRIPTION
**Pull request by shaylipatel**
Exercise 1 project work
- display the list of to-do items
- create new to-do item

### CODE REVIEW

File: todo_app/app.py [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import get_items, add_item
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
-
+@app.route('/items')
 @app.route('/')
```
Comment (JackMeadDev) [inline todo_app/app.py]: Is there a reason you've chosen to give this a second route? No particular reason it's an issue, but all else being equal having only one way to do things means less space for something to go wrong!

File: todo_app/app.py [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
 def index():
-    return 'Hello World!'
+    items = get_items()
+    return render_template('index.html', items=items)
+
+@app.route('/items/new', methods=['POST'])
+def new_item():
+    title = request.form.get('field_name')
```
Comment (JackMeadDev) [inline todo_app/app.py]: `field_name` could probably have a more descriptive name!

**Review by JackMeadDev — APPROVED** [review]
Nice work, I was easily able to run your app & create/view to-dos, and you've got a simple clean solution. 😄 I've left a couple of optional thoughts on the files, but happy for you to merge this when you're ready

File: todo_app/app.py (hunk 1 above)
Comment (shaylipatel) [reply todo_app/app.py]: I was testing it out having multiple routes. Comments are noted and I have updated the project so there is a singular route.

File: todo_app/app.py (hunk 2 above)
Comment (shaylipatel) [reply todo_app/app.py]: I have updated it to title_name

**Pull request merged**